src/kernel-install/kernel-install

```diff
@@ -18,38 +18,31 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with systemd; If not, see <http://www.gnu.org/licenses/>.
 
-SKIP_REMAINING=77
+skip_remaining=77
```

Comment:
Because it's not a global in the actual sense (this changes some shellcheck semantics, and reads better because it doesn't originate from the environment).

Reply:
Yes, but this is a constant, which are traditionally uppercased.

Reply:
It's a variable that originates locally and not from the environment => it's lcase. You'll do well to note /bin/sh doesn't have constants.